src/plugins/vis_types/vega/public/services.ts

```diff
@@ -35,3 +34,6 @@ export const [getDocLinks, setDocLinks] = createGetterSetter<DocLinksStart>('doc
 
 export const [getUsageCollectionStart, setUsageCollectionStart] =
   createGetterSetter<UsageCollectionStart>('UsageCollection');
+
+export const [getThemeStart, setThemeStart] =
+  createGetterSetter<ThemeServiceStart>('ThemeServiceStart');
```

Comment:
can be renamed to `getThemeService`? I don't fully get why is postfixed with `Start` except for the type called `ThemeServiceStart` but I still don't get even this one :)

Reply:
Done https://github.com/elastic/kibana/pull/173715/commits/e3df400c0eab9b24e10041524272f2a1e3f3964b